docs/docs/development.md

```diff
@@ -98,6 +98,18 @@ To install a specific version of Emscripten (e.g. `2.0.6`):
 
 ---
 
+## `perspective-python`
+
+To build the Python library, first configure your project to build Python via
+`yarn setup`. Then, install the requirements corresponding to your version of
+python, e.g.
+
+```bash
+pip install -r python/perspective/requirements-311.txt
+```
+
+`perspective-python` supports Python 3.8 and upwards.
+
```

Comment:
`yarn _requires_python` is the correct way to do this. While this is not documented, a good place to look is always the [CI actions yaml](https://github.com/finos/perspective/blob/master/.github/workflows/build.yml)